packages/x-components/src/x-modules/facets-next/components/lists/__tests__/filters-injection.mixin.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { Filter } from '@empathyco/x-types-next';
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue, { ComponentOptions } from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import {
+  createNextHierarchicalFilter,
+  createNextSimpleFilter
+} from '../../../../../__stubs__/filters-stubs.factory';
+import { XProvide } from '../../../../../components/decorators/injection.decorators';
+import FiltersInjectionMixin from '../filters-injection.mixin';
+
+@Component({
+  template: `
+    <div>
+    <slot/>
+    </div>`
+})
+class Provider extends Vue {
+  @Prop()
+  @XProvide('filters')
+  public injectedData!: any;
+}
+
+const Filters: ComponentOptions<Vue> = {
+  mixins: [FiltersInjectionMixin],
+  template: `
+    <div>
+    <li v-for="filter of renderedFilters" :key="filter.id">{{ filter.id }}</li>
+    </div>
+  `
+};
+
+describe('Test filters injection mixin', () => {
```

Comment:
`describe`s should begin with lowercase

```suggestion
describe('test filters injection mixin', () => {
```

- [ ] `describe` to lowercase